pkg/config/rule_matching.go

```diff
@@ -0,0 +1,100 @@
+package config
+
+import (
+	"math"
+	"regexp"
+	"strconv"
+	"strings"
+)
+
+const (
+	gt = iota
+	eq
+	ge
+	lt
+	le
+)
+
+type RuleMatcher interface {
+	MatchString(s string) bool
+}
+
+func ParseRuleMatcher(label, s string) RuleMatcher {
+	if strings.HasPrefix(s, "~= ") {
+		return regexpMatcher{regexp.MustCompile(s[3:])}
+	} else if strings.HasPrefix(s, "> ") {
+		return newNumberMatcher(gt, s[2:])
+	} else if strings.HasPrefix(s, "= ") {
+		if _, err := strconv.ParseFloat(s[2:], 64); err == nil {
```

Comment:
I think this warrants at least a note in the documentation, that "=" on numbers is not an exact equals, but, due to the nature of floats, is an approximation.